wagtail/admin/templates/wagtailadmin/pages/edit.html

```diff
@@ -57,7 +57,7 @@ <h1 class="left col9 header-title" title="{% blocktrans with title=page.get_admi
             <ul>
                 <li class="footer__container">
                     <nav aria-label="{% trans 'Actions' %}">
-                        <ul>
+                        <ul id="exTest" class="disclosure-nav">
```

Comment:
I might recommend instead of grabbing the `ul` it might be better to base the JS work on the `div.dropdown` as this appears to be the more generic usage of this kind of thing, and is will mean that the JS can work for more cases, not just when it is in a `ul`.

(2 js/keyboard)

Reply:
Also, best to remove this id, it does not appear to be part of the implementation.